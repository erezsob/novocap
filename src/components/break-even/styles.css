.container {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    align-items: center;
    height: 20px;
    width: 90%;
    padding-top: 20px;
    color: white;
    font-weight: bold;
    font-size: 20px;
}

.barContainer {
    margin: 20px;
    width: 150px;
    height: 150px;
    position: relative;
    display: flex;
    flex-direction: column;
    justify-content: center;
}

.title {
    display: flex;
    flex-direction: column;
    padding-top: 13px;
}

.date {
    font-size: 11px;
    font-weight: 300;
}

.addCosts {
    color: #fff;
    font-size: 19px;
}