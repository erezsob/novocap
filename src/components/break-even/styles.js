import { blueMain } from '../../common/variables.css';

export default {
  card: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    boxShadow: 'none',
    height: 208,
    backgroundColor: blueMain,
    cursor: 'pointer'
  },
  iconButton: {
    color: 'white',
    width: 24
  }
};
