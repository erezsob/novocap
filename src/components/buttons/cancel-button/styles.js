import { pinkyRed } from '../../../common/variables.css';

export default {
  cancel: {
    color: pinkyRed
  }
};
