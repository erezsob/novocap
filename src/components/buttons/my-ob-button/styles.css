@value blueLight, bpXSmall from '../../../common/variables.css';

.button {
    border-radius: 4px;
    background-color: blueLight;
    color: white;
    padding: 8px 0;
    font-size: 12px;
    display: table;
    border: 0;
    outline: 0;
    cursor: pointer;
}

.bigger {
    padding: 10px 0;
    font-size: 16px;
}

.icon,
.text {
    vertical-align: middle;
    text-align: center;
}

.text {
    padding: 0 20px;
    font-weight: lighter;
}

.icon ~ .text {
    display: none;
}

.icon {
    display: table-cell;
    padding: 0 10px;
    position: relative;
    top: 2px;
}

@media (min-width: bpXSmall) and (min-device-width: bpXSmall) {
    .icon ~ .text {
        padding-left: 0;
        display: table-cell;
    }
}
