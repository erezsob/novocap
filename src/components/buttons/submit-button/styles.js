import { greenGrass } from '../../../common/variables.css';

export default {
  submit: {
    color: greenGrass
  }
};
