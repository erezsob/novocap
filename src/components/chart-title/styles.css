.label {
    letter-spacing: 0.05em;
    font-size: 20px;
    border-bottom: 1px solid #efefef;
    padding: 20px;
    margin-bottom: 20px;
    font-weight: bold;
    color: #323435;
}