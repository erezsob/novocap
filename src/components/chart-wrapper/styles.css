.container {
    padding: 0 20px 30px;
    position: relative;
    font-size: 12px;
}