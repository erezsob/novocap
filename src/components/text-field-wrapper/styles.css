@value errorText from '../../common/variables.css';

.error {
    font-size: 12px;
    color: errorText
}