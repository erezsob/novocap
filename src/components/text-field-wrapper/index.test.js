import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import TextField from 'material-ui/TextField';
import TextFieldWrapper from './';
import styles from './styles.css';

describe('<TextFieldWrapper />', () => {
  let wrapper;
  const props = {
    label: 'Text Field Label',
    meta: {
      touched: false,
      error: undefined
    }
  };

  beforeEach(() => {
    wrapper = shallow(<TextFieldWrapper {...props} />);
  });

  it('renders', () => {
    expect(wrapper).toHaveLength(1);
  });

  it('renders properly', () => {
    const tree = renderer.create(<TextFieldWrapper {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders by default <TextField /> element with the type of text', () => {
    expect(wrapper.find(TextField).props().type).toBe('text');
  });

  it('renders <TextField /> element with the given type', () => {
    wrapper.setProps({ type: 'fooType' });
    expect(wrapper.find(TextField).props().type).toBe('fooType');
  });

  it('renders a <TextField /> element with the given label', () => {
    expect(wrapper.find(TextField).props().label).toBe(props.label);
  });

  describe('Text Field Error', () => {
    it('renders properly with error message', () => {
      const errorProps = {
        ...props,
        meta: { touched: true, error: 'Required' }
      };
      const tree = renderer
        .create(<TextFieldWrapper {...errorProps} />)
        .toJSON();
      expect(tree).toMatchSnapshot();
    });

    it('renders an error label', () => {
      wrapper.setProps({ meta: { touched: true, error: 'Required' } });
      const errorLabel = wrapper.find(`.${styles.error}`);
      expect(errorLabel).toHaveLength(1);
      expect(errorLabel.text()).toBe('Required');
    });
  });
});
