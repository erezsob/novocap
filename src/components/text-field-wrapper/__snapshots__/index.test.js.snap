// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextFieldWrapper /> Text Field Error renders properly with error message 1`] = `
<div>
  <div
    aria-describedby={undefined}
    className="MuiFormControl-root-1"
    onBlur={[Function]}
    onFocus={[Function]}
  >
    <label
      className="MuiFormLabel-root-11 MuiFormLabel-error-14 MuiInputLabel-root-5 MuiInputLabel-formControl-6 MuiInputLabel-animated-9"
      data-shrink={false}
      htmlFor={undefined}
    >
      Text Field Label
    </label>
    <div
      className="MuiInput-root-17 MuiInput-error-22 MuiInput-formControl-18 MuiInput-underline-21"
    >
      <input
        aria-invalid={true}
        aria-required={false}
        autoComplete={undefined}
        autoFocus={undefined}
        className="MuiInput-input-25"
        defaultValue={undefined}
        disabled={false}
        id={undefined}
        name={undefined}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={undefined}
        onKeyUp={undefined}
        placeholder={undefined}
        readOnly={undefined}
        required={undefined}
        rows={undefined}
        type="text"
        value={undefined}
      />
    </div>
  </div>
  <span
    className="error"
  >
    Required
  </span>
</div>
`;

exports[`<TextFieldWrapper /> renders properly 1`] = `
<div>
  <div
    aria-describedby={undefined}
    className="MuiFormControl-root-1"
    onBlur={[Function]}
    onFocus={[Function]}
  >
    <label
      className="MuiFormLabel-root-11 MuiInputLabel-root-5 MuiInputLabel-formControl-6 MuiInputLabel-animated-9"
      data-shrink={false}
      htmlFor={undefined}
    >
      Text Field Label
    </label>
    <div
      className="MuiInput-root-17 MuiInput-formControl-18 MuiInput-underline-21"
    >
      <input
        aria-invalid={false}
        aria-required={false}
        autoComplete={undefined}
        autoFocus={undefined}
        className="MuiInput-input-25"
        defaultValue={undefined}
        disabled={false}
        id={undefined}
        name={undefined}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onKeyDown={undefined}
        onKeyUp={undefined}
        placeholder={undefined}
        readOnly={undefined}
        required={undefined}
        rows={undefined}
        type="text"
        value={undefined}
      />
    </div>
  </div>
</div>
`;
